src/content/docs/resources/placeholders.mdx

```diff
@@ -0,0 +1,532 @@
+---
+title: Placeholders file
+description: Placeholders file
+tableOfContents: false
+---
+
+
+The `placeholders` file is the source for storefront labels across drop-in components. The `placeholders` file has two columns: **Keys** and **Values**. The Keys column contains the text variables that are replaced at runtime. The Values column contains the text values that replace the variables. This file provides a quick and easy way for content creators and merchants to make changes to the site's labels without developer involvement.
+
+## Placeholders default values
+
+The initial values in the `placeholder` file within the boilerplate's starter content provides the default out-of-the-box labels for the drop-in components.
+
+| **Key**                                                                                                         | **Value**                                                                                                                                                                                    |
```

Comment:
Should "key" and "value" be plural here? On line 8, they're plural.

Reply:
Nope. The other way around. Good catch.